fuzzers/libfuzzer_libpng_launcher/src/lib.rs

```diff
@@ -202,7 +202,7 @@ pub fn libafl_main() {
         };
 
         // Create the executor for an in-process function with one observer for edge coverage and one for the execution time
-        let mut executor = TimeoutExecutor::new(
+        let mut executor = TimeoutExecutor::batch_mode(
```

Comment:
I'd just keep it at `new` (and maybe add an extra non-batched thing here), so that all users just get the better perf for free

Reply:
let's see if it is not buggy before and eventually change later